src/components/ShopDropdownMenu/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { ShopDropdownMenuProps } from '@/types';
+import s from './ShopDropdownMenu.module.scss';
+import { storePanelButtonsList } from '@/constants/storePanelButtonsList';
+import { NavLink } from 'react-router-dom';
+
+export const ShopDropdownMenu = ({ setDropdownOpen }: ShopDropdownMenuProps) => {
+
+  
+  return (
+    <div className={s.shopDropdown}>
+      {true ? (
```

Comment:
what do you mean by true here, if this component will render only if the user clicks on it?

Reply:
i left it for future check of user having a shop, but i think it will be done in header component, ill change it

Reply:
is better to leave it empty then, because there is no check as such, because true will always return true